app/javascript/app/components/table/table.js

```diff
@@ -45,10 +45,12 @@ class TableContainer extends PureComponent {
   };
 
   setColumnWidth = column => {
-    if (column === 'year') {
+    const narrowColumns = ['year', 'unit', 'first', 'last'];
```

Comment:
Maybe we can move columns names to a constant as data used to set rows height in `setRowsHeight` function?